posix/eventfd.c

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// clang-format off
+#include <openenclave/enclave.h>
+// clang-format on
+
+#include <openenclave/internal/thread.h>
+#include <openenclave/corelibc/unistd.h>
+#include <openenclave/internal/utils.h>
+#include <openenclave/internal/print.h>
+#include <openenclave/internal/trace.h>
+#include <openenclave/corelibc/sys/eventfd.h>
+#include <openenclave/internal/posix/fdtable.h>
+#include <openenclave/internal/posix/device.h>
+#include <openenclave/internal/posix/raise.h>
+
+int oe_eventfd(unsigned int initval, int flags)
```

Comment:
>oe_eventfd [](start = 4, length = 10)

For my understanding, what is supposed to use eventfd in the upcoming changesets? Not quite sure how it fits in with the filesystem/sockets work.

Reply:
Removed eventfd feature from this PR.